reference/docs-conceptual/community/contributing/powershell-style-guide.md

```diff
@@ -0,0 +1,492 @@
+---
+title: PowerShell-Docs style guide
+description: This article provides the rules of style for writing PowerShell documentation.
+ms.date: 03/05/2020
+ms.topic: conceptual
+---
+# PowerShell-Docs style guide
+
+This article provides style guidance specific to the PowerShell-Docs content. This builds on the
+information outlined in the [Overview](overview.md#get-started-writing-docs).
+
+## Product Terminology
+
+There are several variants of PowerShell.
+This table defines some of the different terms used to discuss PowerShell.
+
+- **PowerShell** - This is the default. We consider PowerShell 7 and beyond to be the one, true
+  PowerShell going forward.
+
+- **PowerShell Core** - PowerShell built on .NET Core. Usage of the term **Core** should be limited
+  to cases where it is necessary to differentiate it from Windows PowerShell.
+
+- **Windows PowerShell** - PowerShell built on .NET Framework. Windows PowerShell ships only on
+  Windows and requires the complete Framework.
+
+In general, references to "Windows PowerShell" in documentation can be changed to "PowerShell".
+"Windows PowerShell" should **not** be changed when Windows-specific technology is being discussed.
+
+## Markdown specifics
+
+The Microsoft Open Publishing System (OPS) that builds our documentation uses [markdig][] to process
+the Markdown documents. Markdig parses the documents based on the rules of the latest [CommonMark][]
+specification.
+
+The new CommonMark spec is much stricter about the construction of some Markdown elements. Pay close
+attention to the details provided in this document.
+
+### Blank lines, spaces, and tabs
+
+Remove duplicate blank lines. Multiple blank lines render as a single blank line in HTML so there is
+not purpose for multiple blank lines.
+
+Blank lines also signal the end of a block in Markdown. There should be a single blank between
+Markdown blocks of different types (for example, between a paragraph and a list or header).
+
+> [!NOTE]
+> Spacing is significant in Markdown. Always uses spaces instead of hard tabs. Remove
+> extra spaces at the end of lines.
+
+### Titles and headings
+
+Only use [ATX headings][atx] (# style, as opposed to `=` or `-` style headers).
+
+- Use sentence case - only proper nouns and the first letter of a title or header should be
+  capitalized
+- There must be a single space between the `#` and the first letter of the heading
+- Headings should be surrounded by a single blank line
+- Only one H1 per document
+- Header levels should increment by one. Do not skip levels
```

Comment:
This is the skip comment I was looking for in the other doc.  This should probably be added there.